zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -350,13 +348,25 @@ where
                                 data: None,
                             })?;
 
-            let chain_info = match response {
+            let GetBlockTemplateChainInfo {
+                tip_height,
+                tip_hash,
+                expected_difficulty,
+                cur_time,
+                min_time,
+                max_time,
+                history_tree
+            } = match response {
```

Comment:
This will probably conflict with the refactor, because we'll be returning all the fetched data from a single function.